README.md

```diff
@@ -33,7 +22,8 @@ module.exports = function(config) {
 
     // the default configuration
     allureReport: {
-      reportDir: '',
+      reportDir: '', // By default files will be save in the base dir
+      useBrowserName: true // add browser name to report and classes names
```

Comment:
Normally, new features are introduced in a backward-compatible way, so the default behavior should stay as it was before.

So, it should be false by default

Reply:
Sorry for the late replay, I've been out for Christmas holidays 🎄

I don't know if I misunderstand something but as I see here, https://github.com/allure-framework/allure-karma/blob/8f17600848a72d21412a7852a902c996483bad87/src/AllureReporter.js#L65 I understand that the default behavior adds `browser.name` to the `suiteName` that is why I add `true` as the default value for `useBrowserName`. 😅

The main goal of this PR is just add the option to remove the `browser.name` from the `suiteName` 😉

If you think I misunderstand something please let me know 👍

Reply:
You are right! I did not work with this code for a while, so I forgot how is this was supposed to work.